ignite/contrib/handlers/lr_finder.py

```diff
@@ -0,0 +1,366 @@
+# coding: utf-8
+import logging
+import warnings
+from collections.abc import Mapping
+import tempfile
+import contextlib
+from pathlib import Path
+
+import torch
+from torch.optim.lr_scheduler import _LRScheduler
+
+from ignite.engine import Events, Engine
+from ignite.handlers import Checkpoint
+from ignite.contrib.handlers import LRScheduler, PiecewiseLinear
+
+
+class FastaiLRFinder(object):
+    """Learning rate finder handler for supervised trainers.
+
+    While attached, the handler increases the learning rate in between two
+    boundaries in a linear or exponential manner. It provides valuable
+    information on how well the network can be trained over a range of learning
+    rates and what can be an optimal learning rate.
+
+    Examples:
+
+    .. code-block:: python
+
+        from ignite.contrib.handlers import LRFinder
```

Comment:
This is inconsistent with the actual class name of `FastaiLRFinder`

Reply:
Nice catch, will fix!